source/examples/rapids-snowflake-cudf/notebook.ipynb

```diff
@@ -0,0 +1,356 @@
+{
```

Comment:
<p>I would use the <a href="https://github.com/rapidsai/deployment?tab=readme-ov-file#docref" target="_blank">docref</a> admonition int he first paragraph to cross reference the Snowflake page. This has a few benefits including inserting a link back to this example at the bottom of the snowflake page.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/deployment/pull/505/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>